ledger/catchpointtracker.go

```diff
@@ -424,6 +527,35 @@ func (ct *catchpointTracker) commitRound(ctx context.Context, tx *sql.Tx, dcc *d
 		return err
 	}
 
+	if dcc.catchpointFirstStage {
+		err = writeCatchpointStateUint64(ctx, tx, catchpointStateWritingFirstStageInfo, 1)
+		if err != nil {
+			return err
+		}
+	}
+
+	err = writeCatchpointStateUint64(
```

Comment:
shouldn't it be under dcc.catchpointFirstStage or `ct.catchpointInterval != 0 && len(rounds) > 0`

Reply:
yes but exceed db write